packages/commercetools/composables/src/getters/orderGetters.ts

```diff
@@ -1,26 +1,46 @@
-import { UserOrderGetters, AgnosticOrderStatus } from '@vue-storefront/interfaces';
-import { Order, OrderState } from './../types/GraphQL';
+import { UserOrderGetters, AgnosticOrderStatus, AgnosticPrice } from '@vue-storefront/interfaces';
+import { Order, OrderState, LineItem, Money, Address } from './../types/GraphQL';
 
 export const getOrderDate = (order: Order): string => order?.createdAt || '';
 
 export const getOrderId = (order: Order): string => order?.id || '';
 
+export const getOrderNumber = (order: Order): string => order?.orderNumber || getOrderId(order);
+
 const orderStatusMap = {
   [OrderState.Open]: AgnosticOrderStatus.Open,
   [OrderState.Confirmed]: AgnosticOrderStatus.Confirmed,
   [OrderState.Complete]: AgnosticOrderStatus.Complete,
   [OrderState.Cancelled]: AgnosticOrderStatus.Cancelled
 };
 
+const getPrice = (money: Money): AgnosticPrice => ({
+  regular: money?.centAmount ? money.centAmount / 100 : 0
+});
+
 export const getOrderStatus = (order: Order): AgnosticOrderStatus | '' => order?.orderState ? orderStatusMap[order.orderState] : '';
 
-export const getOrderPrice = (order: Order): number | null => order ? order.totalPrice.centAmount / 100 : null;
+export const getOrderItems = (order: Order): LineItem[] => order?.lineItems || [];
+
+export const getOrderPrice = (order: Order): AgnosticPrice => getPrice(order?.totalPrice);
+
+const transformAddressToString = (address: Address): string => (
```

Comment:
shouldn't that be a set of helpers OR just a non-agnostic object?

@andrzejewsky we've just discussed that

Reply:
It's temporary actually (just to keep DRY)

Reply:
if something is temporarly add an issue and todo here with issue linked